cmd/kube-controller-manager/app/plugins.go

```diff
@@ -95,32 +97,18 @@ func ProbeRecyclableVolumePlugins(config componentconfig.VolumeConfiguration) []
 	}
 	allPlugins = append(allPlugins, nfs.ProbeVolumePlugins(nfsConfig)...)
```

Comment:
This approach was turned down in PR https://github.com/kubernetes/kubernetes/pull/21552